specification/vi/resource-manager/Microsoft.VideoIndexer/preview/2021-04-01-preview/vi.json

```diff
@@ -0,0 +1,844 @@
+          "readOnly": false,
+          "type": "string",
+          "description": "The requested permission",
+          "enum": [
+            "Contributor",
+            "Reader"
+          ],
+          "x-ms-enum": {
+            "name": "Contributor",
+            "modelAsString": true
+          }
+        },
+        "scope": {
+          "readOnly": false,
+          "type": "string",
+          "description": "The requested media type",
+          "enum": [
+            "Video",
+            "Account",
+            "Project"
+          ],
+          "x-ms-enum": {
+            "name": "Video",
+            "modelAsString": true
+          }
+        },
+        "videoId": {
+          "description": "The video ID",
+          "type": "string",
+          "readOnly": false,
+          "example": "07ec9e38d4"
+        },
+        "projectId": {
+          "description": "The project ID",
+          "type": "string",
+          "readOnly": false,
+          "example": ""
+        }
+      }
+    },
+    "MediaServices": {
+      "description": "The media services details",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "The media services resource id",
+          "readOnly": false,
+          "example": "/subscriptions/xxx/resourceGroups/contoso-videoanalyzer-rg/providers/Microsoft.Media/mediaservices/contoso-videoanalyzer-ms"
+        },
+        "userAssignedManagedIdentity": {
+          "type": "object",
+          "description": "The user assigned managed identity resource ID when connecting to media services resource",
+          "readOnly": false,
+          "$ref": "#/definitions/UserAssignedManagedIdentity"
+        }
+      }
+    },
+    "UserAssignedManagedIdentity": {
+      "description": "The user assigned managed identity details",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "The user assigned managed identity resource id",
+          "readOnly": false,
+          "example": "/subscriptions/xxx/resourceGroups/contoso-videoanalyzer-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/contoso-videoanalyzer-mi"
+        }
+      }
+    },
+    "AccessToken": {
+      "description": "Azure Video Analyzer for Media access token.",
+      "type": "object",
+      "properties": {
+        "accessToken": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The access token.",
+          "x-ms-secret": true,
+          "example": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJPbmxpbmUgSldUIEJ1aWxkZXIiLCJpYXQiOjE2MjI0NDc3MzMsImV4cCI6MTY1Mzk4MzczMywiYXVkIjoid3d3LmV4YW1wbGUuY29tIiwic3ViIjoianJvY2tldEBleGFtcGxlLmNvbSIsIkdpdmVuTmFtZSI6IkpvaG5ueSIsIlN1cm5hbWUiOiJSb2NrZXQiLCJFbWFpbCI6Impyb2NrZXRAZXhhbXBsZS5jb20iLCJSb2xlIjpbIk1hbmFnZXIiLCJQcm9qZWN0IEFkbWluaXN0cmF0b3IiXX0"
+        }
+      }
+    },
+    "Operation": {
```

Comment:
nit: You can refer operation definition from /specification/common-types/resource-management/v2/types.json#L338